test/src/test/java/hudson/triggers/TriggerStartTest.java

```diff
@@ -95,10 +93,14 @@
         p.addTrigger(t);
         p.save();
         p = j.jenkins.copy(p, "nue");
-        t = p.getTrigger(MockTrigger.class);
-        assertNotNull(t);
-        assertEquals("[true]", t.calls.toString());
+        extractedMethod95320(p); // CAP AL
     }
+ // CAP AL
+    private void extractedMethod95320(final AbstractProject p) { // CAP AL
+        MockTrigger t = p.getTrigger(MockTrigger.class); // CAP AL
```

Comment:
Not compilable: 

![image](https://user-images.githubusercontent.com/21219182/115799995-3a52cd80-a40c-11eb-8617-21f7ffc72e32.png)

The method parameter should have been declared as a generic type 

```java
private void extractedMethod95320(final AbstractProject<?,?> p) { // CAP AL
    MockTrigger t = p.getTrigger(MockTrigger.class); // CAP AL
```